Create zh_CN.lang

SihenZhang opened the pull request.

SihenZhang committed:
Create zh_CN.lang

SihenZhang committed:
Merge remote-tracking branch 'refs/remotes/Bartz24/1.12-2.0' into 1.12-2.0

SihenZhang committed:
Update zh_CN.lang

Bartz24 commented:
I've got a couple more things to add, and then I'll tell you I'm ready to merge it just to make sure all the translations are added :D

Bartz24 commented:
The lang files should be stable now for the next version, so you can make the changes for the new stuff now

SihenZhang committed:
Merge remote-tracking branch 'refs/remotes/Bartz24/1.12-2.0' into 1.12-2.0

SihenZhang committed:
Merge remote-tracking branch 'refs/remotes/Bartz24/1.12-2.0' into 1.12-2.0

SihenZhang committed:
Merge remote-tracking branch 'refs/remotes/Bartz24/1.12-2.0' into 1.12-2.0

SihenZhang committed:
Update zh_CN.lang

The improved end portal need 16 purpur pillar but not 12, so I fix it in
en_US.lang

SihenZhang commented:
I think now I finished the language file
